<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_QuickInfoWidget</class>
 <widget class="QWidget" name="LC_QuickInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layToFlexible">
         <item>
          <widget class="QToolButton" name="tbSelectEntity">
           <property name="toolTip">
            <string>Select entity for properties</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/extui/extui.qrc">
             <normaloff>:/extui/menuselect.png</normaloff>:/extui/menuselect.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbFind">
           <property name="toolTip">
            <string>Select current entity in drawing</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/icons/icons.qrc">
             <normaloff>:/icons/select_entity.svg</normaloff>:/icons/select_entity.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbEditProperties">
           <property name="toolTip">
            <string>Properties Edit</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/ui/ui.qrc">
             <normaloff>:/ui/blockedit.png</normaloff>:/ui/blockedit.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbCopy">
           <property name="toolTip">
            <string>Copy to Clipboard</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/icons/icons.qrc">
             <normaloff>:/icons/copy.svg</normaloff>:/icons/copy.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbClear">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/main/main.qrc">
             <normaloff>:/main/editclear.png</normaloff>:/main/editclear.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbCollectCoords">
           <property name="toolTip">
            <string>Collect Coordinates</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/extui/extui.qrc">
             <normaloff>:/extui/polylineadd.png</normaloff>:/extui/polylineadd.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCoordinatesMode">
           <property name="text">
            <string>Coordinates:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbPointsCoordinatesMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Mode to display collected coordinates</string>
           </property>
           <item>
            <property name="text">
             <string>Absolute</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Relative</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Delta to First</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Delta to Previous</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tbSettings">
           <property name="toolTip">
            <string>Settings</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/icons/icons.qrc">
             <normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="LC_PlainTextEdit" name="pteInfo" native="true">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
       <property name="markdown" stdset="0">
        <string/>
       </property>
       <property name="html" stdset="0">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="pteInfo1">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC_PlainTextEdit</class>
   <extends>QWidget</extends>
   <header>lc_plaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../res/extui/extui.qrc"/>
  <include location="../../../../res/main/main.qrc"/>
  <include location="../../../../res/icons/icons.qrc"/>
  <include location="../../../../res/icons/icons.qrc"/>
  <include location="../../../../res/ui/ui.qrc"/>
 </resources>
 <connections/>
</ui>
